src/modules/components/vote/voteProposalDataListItem/voteProposalDataListItemSkeleton/voteProposalDataListItemSkeleton.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { VoteProposalDataListItem } from '..';
+import { DataList } from '../../../../../core';
+
+const meta: Meta<typeof VoteProposalDataListItem.Skeleton> = {
+    title: 'Modules/Components/Vote/VoteProposalDataListItem/VoteProposalDataListItem.Skeleton',
+    component: VoteProposalDataListItem.Skeleton,
+    tags: ['autodocs'],
```

Comment:
This is not needed anymore after the latest changes introduced by @rcaragon with the Storybook v8 update
```suggestion
```